Simplify README and add cloud deployment (#7906)
docker/compose: README.md

Docker Compose
==============
[![Build Status](https://ci-next.docker.com/public/buildStatus/icon?job=compose/master)](https://ci-next.docker.com/public/job/compose/job/master/)

![Docker Compose](logo.png?raw=true "Docker Compose Logo")

Docker Compose is a tool for running multi-container applications on Docker
defined using the [Compose file format](https://compose-spec.io).

> A Compose file is used to define how the one or more containers that make up

Reviewer: ```suggestion
A Compose file is used to define how the container(s) that make up
```

Author: I wanted to call out one container here as we see Compose as a way to script `docker run` as well.

your application are configured.
Once you have a Compose file, you can create and start your application with a
single command: `docker-compose up`.

> Compose files can be used to deploy applications to

Reviewer: ```suggestion
Compose files can be used to deploy applications locally, or to
```

Author: Good catch, will fix.
